Configure.ps1

```diff
@@ -84,25 +85,32 @@ so that they are consistent with the Applications parameters
     # $tenantId is the Active Directory Tenant. This is a GUID which represents the "Directory ID" of the AzureAD tenant 
     # into which you want to create the apps. Look it up in the Azure portal in the "Properties" of the Azure AD. 
 
-    # Login to Azure PowerShell (interactive: you'll need to sign-in with creds enabling your to create apps in the tenant)
-    $creds = Connect-AzureAD -TenantId $tenantId
+    # Login to Azure PowerShell (interactive if credentials are not already provided: 
+    # you'll need to sign-in with creds enabling your to create apps in the tenant)
+    if (!$Credential)
```

Comment:
There seems to be quite a lot of duplicated code. You could create a file with functions that encapsulate the duplicated code.